src/NServiceBus.TransactionalSession.AcceptanceTests/TestingInfrastructure/TransactionSessionDefaultServer.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace NServiceBus.TransactionalSession.AcceptanceTests;
+
+using System;
+using System.Threading.Tasks;
+using NServiceBus.AcceptanceTesting.Support;
+using Persistence.CosmosDB.TransactionalSession;
+
+public class TransactionSessionDefaultServer : DefaultServer
+{
+    public static Action<EndpointConfiguration> ConfigurePersistence { get; set; } =
+        _ => throw new NotImplementedException();
+
+    public override Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor,
+        EndpointCustomizationConfiguration endpointConfiguration,
+        Action<EndpointConfiguration> configurationBuilderCustomization)
+    {
+        endpointConfiguration.TypesToInclude.Add(typeof(CaptureBuilderFeature));
+        endpointConfiguration.TypesToInclude.Add(typeof(CosmosDbTransactionalSessionFeature));
```

Comment:
That's the thing I don't understand why I need to do this currently

Reply:
all types from the acceptance test project itself are excluded from scanning by default (https://github.com/Particular/NServiceBus.Persistence.CosmosDB/pull/415/files#diff-49ec7e99da19569a2afdbe8174060351275338bfc3419aa4dc6cfdcd4da77efdR22) and `Features` can only be activated if they are scanned, explicitly enabling them via the `EnableFeature` API doesn't suffice (raised an issue for that already)